locale/ca_ES/default.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+
+<!--
+  * locale/ca_ES/default.xml
+  *
+  * Copyright (c) 2014-2015 Simon Fraser University Library
```

Comment:
Everything is set to the copyright dates in the corresponding en_US locale files. Couldn't we just update all copyright dates in all files after merging these changes?